rust_src/src/lib.rs

```diff
@@ -151,6 +151,7 @@ pub extern "C" fn rust_init_syms() {
         defsubr(&*windows::Swindowp);
         defsubr(&*windows::Swindow_live_p);
         defsubr(&*process::Sget_process);
+        defsubr(&*process::Sprocessp);
```

Comment:
Do you mean a `pub use` in `lib.rs`? Not if the build succeeds, which means it's not used from C.